services/templates.ts

```diff
@@ -177,45 +178,51 @@ const parseTemplatesForHighLowPrice = async (
  * Gets the lowest price of assets for sale in a list of templates
  * Mostly used to display the lowest price of any of the templates with assets for sale in the collection
  * @param  {Template[]} allTemplates   An array of templates you want to look up the lowest price for
+ * @param  {string} collection         Name of collection that templates belong to
  * @return {Template[]}                Returns array of templates with an additional 'lowestPrice' flag
  */
 
-const parseTemplatesForLowPrice = async (
-  allTemplates: Template[]
+const parseTEmplatesForLowPrice = async (
+  allTemplates: Template[],
+  collection: string
 ): Promise<Template[]> => {
-  const templateIdsByPrice = {};
+  const templateIdsByLowestPrice = {};
 
-  await asyncForEach(allTemplates, async (template: Template) => {
-    const res = await templateSalesApiService.getAll({
-      collection_name: template.collection.collection_name,
-      template_id: template.template_id,
-      symbol: TOKEN_SYMBOL,
-      sort: 'price',
-      order: 'asc',
-      limit: '1',
-    });
+  const results = await templateSalesApiService.getAll({
+    symbol: TOKEN_SYMBOL,
+    collection_name: collection,
+    order: 'desc',
+    sort: 'created',
+  });
+  const precision = results?.data[0]?.price.token_precision;
```

Comment:
If `results.data` is an empty array, let's just do an early return so that the loops below don't need to run.

```js
    return {
      ...template,
      lowestPrice: '',
    };
```